src/main/webapp/js/feed-loader.js

```diff
@@ -1,3 +1,21 @@
+
+
+/**
+ * Shows the message form if the user is logged in.
+ */
+ function showMessageFormIfLoggedIn() {
+   fetch('/login-status')
+    .then((response) => {
+      return response.json();
```

Comment:
Nit: this can be replaced by .then((response) => response.json())

Reply:
ok!